app/models/user.rb

```diff
@@ -762,7 +762,14 @@ def refresh_counts!
           post_count: Post.for_user(id).count,
           post_deleted_count: Post.for_user(id).deleted.count,
           post_update_count: PostVersion.for_user(id).count,
-          note_count: NoteVersion.where(updater_id: id).count
+          favorite_count: Favorite.for_user(id).count,
+          wiki_edit_count: WikiPageVersion.for_user(id).count,
+          note_count: NoteVersion.for_user(id).count,
```

Comment:
Yeah, I don't see a point in the edit ones being here

Reply:
Changed to be just add favorites (as those are known to get out of sync), and replacements